### DISCUSSION TITLE
3.1.1版本,修改logback-spring.xml的log.base后页面读取日志超时

### DISCUSSION THREAD

**Discussion by q4q5q6qw** [apache/dolphinscheduler#14997] [Q&A]
在页面上看日志路径打印出来了，但是读取超时。根据路径在机器上可以找到正常的日志文件，权限也是够的
[LOG-PATH]: /opt/cloud/logs/csb/apache-dolphinscheduler/worker-server/logs/20231009/11177021802944_2-3755-12232.log, [HOST]:  Host{address='0.0.0.0:1234', ip='0.0.0.0', port=1234}
Roll view log error: wait response on the channel Host{address='0.0.0.0:1234', ip='0.0.0.0', port=1234} timeout 10000

**Comment by SJL03** [comment]
你好，这个问题你解决了吗
